### DISCUSSION TITLE
Playground, are you thinking of implementing it?

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [rebing/graphql-laravel#1028] [General]
I have already used Apollo GraphQL in some tests, it has a sensational Playground, I think the package should have something of this level and also an improvement in the documentation both in more detail and covering more cases of errors, which I see is opening a lot of issue and discussion about it.

**Comment by mfn** [comment]
> I have already used Apollo GraphQL in some tests, it has a sensational Playground, I think the package should have something of this level 

The current stable 8.x actually includes GraphQL, see https://github.com/rebing/graphql-laravel/tree/8.x#graphiql .

But this is on its way out in favour of another page which provides a better experience, see https://github.com/rebing/graphql-laravel/pull/986 for the removal (this graphql-laravel version is not released yet, we're on 9.0.0-RC1 only) and https://github.com/mll-lab/laravel-graphiql is the recommended alternative.

> and also an improvement in the documentation both in more detail and covering more cases of errors, which I see is opening a lot of issue and discussion about it.

Absolute agree! Whilst the documentation covers a lot of cases, I realize there is a vast of improvements for clarity and other stuff.

PRs definitely welcome (but not against 8.x anymore, only master please!)
